SConstruct

```diff
@@ -13,12 +13,37 @@ env.Append(CPPPATH=[
     "#thirdparty/openxr/include/",
     ])
 
-common_objects = []
-common_objects.append(env.SharedObject(Glob("#common/src/main/cpp/export/*.cpp")))
-
-SConscript([
-    "godotopenxrmeta/SConstruct",
-    "godotopenxrpico/SConstruct",
-    "godotopenxrlynx/SConstruct",
-    "godotopenxrkhronos/SConstruct",
-    ], 'env common_objects')
+sources = []
+sources += Glob("#common/src/main/cpp/*.cpp")
+sources += Glob("#common/src/main/cpp/export/*.cpp")
+sources += Glob("#common/src/main/cpp/extensions/*.cpp")
+
+binary_path = '#demo/addons/godotopenxrvendors/.bin'
+project_name = 'godotopenxr'
```

Comment:
Should this be `godotopenxrvendors`? That makes it different than the addon name, and could also deceptively seem like it's providing all OpenXR support.

Reply:
Good point, I'll update the name!